datadog_checks_dev/setup.py

```diff
@@ -29,6 +29,7 @@
     'pytest-benchmark',
     'pytest-cov',
     'pytest-mock',
+    'requests>=2.20.0',
```

Comment:
we should pin this to the same version as the base pacakge, python version resolution is not great and it would be bad if requests was upgraded beyond what it should be

Reply:
this is fine b/c base will always override

Reply:
Can you address why that's true?

Reply:
1. base installed first, this requirement already satisfied
2. this installed first, base pin will overwrite whatever this gave